tagbase_server/tagbase_server/utils/processing_utils.py

```diff
@@ -411,34 +425,22 @@ def process_etuff_file(file, version=None, notes=None):
                 )
                 return 1
 
-            insert_new_submission(
-                cur,
-                tag_id,
-                submission_filename,
-                notes,
-                version,
-                entire_file_hash,
-                dataset_id,
-                metadata_hash,
-                content_hash,
-            )
-            submission_id = get_current_submission_id(cur)
-            metadata = [[str(submission_id)]+list(tup) for tup in metadata]
+            # metadata = [[str(submission_id)]+list(tup) for tup in metadata]
 
             proc_obs = []
             variable_lookup = {}
             # at this point we have already read form the file all global attribute lines
-            line_counter = number_global_atttributes_lines
+            line_counter = number_global_attributes_lines
 
-            # TODO we should use the content in the following
+            # TODO we should use the 'content' variable in the following
             s_time = time.perf_counter()
             with open(file, "rb") as data:
```

Comment:
I really don't think we need to do this lines here anymore because we have all of the content in the `content` variable at this point, so no real need for reading again the entire file